src/watson/registration.py

```diff
@@ -52,6 +52,12 @@ def _resolve_field(self, obj, name):
         """Resolves the content of the given model field."""
         name_parts = name.split("__", 1)
         prefix = name_parts[0]
+        # Check if the attribute exists in the database
+        try:
+            if not getattr(obj, prefix):
```

Comment:
BAM! It worked! Thank you very much for this fixing, it saves me about 1 hour a day :D 
Can you upload a new version von django-watson on pypi so I can use the pypi version?

Again, thank you very very much! :)